### ISSUE TITLE
Add a Piwigo Share Extension to streamline uploading images to Piwigo from native Photos app

### ISSUE THREAD

**Issue by pokmis** [Piwigo/Piwigo-Mobile#380]
**Is your feature request related to a problem? Please describe.**
I frequently "mark up" / "touch up" photos in the native iOS app and struggle to locate them quickly when adding them to a Piwigo album.

**Describe the solution you'd like**
It'd be great to have a "Piwigo" share extension that's directly accessible from a selected photo (or set of photos) that allows quick upload to a Piwigo album.

Current Flow
- Open Piwigo
- Select an Album
- Click "Add"
- Navigate / Select photo(s)
- Click "Upload"
- Review / Click "Upload"

Suggested Alternative Flow (Using share extension)
- Open Native iOS Photos App
- Select a photo / set of photos
- Click Share and select Piwigo
- Piwigo prompts user to select album
- Review / Click "Upload"

(I started prototyping the idea, it could look something like this...)
![image](https://user-images.githubusercontent.com/5696872/88244946-53eb9c80-cc63-11ea-9f3f-82b5fab3500f.png)

**Comment by EddyLB** [comment]
Yes, we clearly need a Piwigo Share Extension. This will come after the recoding of the upload management.
Duplicate of #284.

**Comment by EddyLB** [comment]
To be implemented when albums will be stored in a persistent database.

**Comment by EddyLB** [comment]
Fixed in [v4.3](https://github.com/Piwigo/Piwigo-Mobile/pull/658)
